PocketKit/Sources/PocketKit/Navigation/NavigationSidebarViewController.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+import Combine
+import Textile
+
+
+private extension Style {
+    static let title: Style = .header.sansSerif.h7
+}
+
+class NavigationSidebarViewController: UIViewController {
+    private static let layout: UICollectionViewLayout = {
+        var configuration = UICollectionLayoutListConfiguration(appearance: .insetGrouped)
+        configuration.headerMode = .none
+        configuration.backgroundColor = UIColor(.ui.white1)
+        return UICollectionViewCompositionalLayout.list(using: configuration)
+    }()
+    
+    private static let snapshot: NSDiffableDataSourceSnapshot<MainViewModel.AppSection, String> = {
+        var snapshot = NSDiffableDataSourceSnapshot<MainViewModel.AppSection, String>()
+        snapshot.appendSections(MainViewModel.AppSection.allCases)
+        MainViewModel.AppSection.allCases.forEach {
+            snapshot.appendItems([$0.navigationTitle], toSection: $0)
+        }
+        return snapshot
```

Comment:
Let's not conflate the term "app section" with "collection view section" here. This collection only needs a single section with an item for each app section.

Something like this should be all you need:
```swift
var snapshot = NSDiffableDataSourceSnapshot<String, MainViewModel.AppSection>()
let section = "app-sections"
snapshot.appendSections([section])
snapshot.appendItems(MainViewModel.AppSection.allCases, toSection: section)
```

Reply:
@jacobsimeon Fair statement. My initial thought behind multiple collection view sections for each app section was the possibility of this type of sidebar design:

![image](https://user-images.githubusercontent.com/1158092/135475626-7c31a4de-8ce5-4bbe-b98d-cd4febe16b5d.png)

Which, in the current state of this project, results in:

![image](https://user-images.githubusercontent.com/1158092/135475700-6a2cc159-7272-4dbf-b408-8171fa1d8c93.png)

This could probably be modified to use one section with item spacing, however at the time, this did seem more like an inset grouped layout, and letting the system handle the rest.